_topic_map.yml

```diff
@@ -166,9 +166,11 @@ Name: Updating clusters
 Dir: updating
 Distros: openshift-origin,openshift-enterprise
 Topics:
-- Name: Updating a cluster to a minor version from the web console
+- Name: Updating a cluster between minor versions
+  File: updating-cluster-between-minor
```

Comment:
i think this definition about minor version and within a minor version need to be unified.
in v4.1(https://docs.openshift.com/container-platform/4.1/updating/updating-cluster.html), minor version upgrade means 4.1.0-4.1.2,
but in this pr, seems minor version upgrade refer to 4.1-4.2.

Reply:
I'll open a separate PR to fix that language in 4.1.

Reply:
Here's the 4.1 only PR to change these titles: https://github.com/openshift/openshift-docs/pull/17631